splunk-quarkus/src/main/resources/application.properties

```diff
@@ -50,6 +50,9 @@ camel.component.kafka.security-protocol = PLAINTEXT
 camel.component.kafka.ssl-truststore-location =
 camel.component.kafka.ssl-truststore-type = JKS
 camel.component.kafka.max-poll-records = 300
+# https://camel.apache.org/manual/camel-3x-upgrade-guide-3_17.html#_camel_kafka
+camel.component.kafka.allow-manual-commit = true
+camel.component.kafka.kafka-manual-commit-factory = #class:org.apache.camel.component.kafka.consumer.DefaultKafkaManualCommitFactory
```

Comment:
Is the `#` intended before the property value?

Reply:
This is a Camel thingy that converts it to a class instance. I was unable to find documentation about it, but I've found the actual code that handles this. https://github.com/apache/camel/blob/d5204c5ab8c967171fe3c7e9594dbd9a6ebf4af0/core/camel-support/src/main/java/org/apache/camel/support/EndpointHelper.java#L312-L323